crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -651,6 +652,9 @@ impl SpecializedMeshPipeline for MeshPipeline {
         let mut shader_defs = Vec::new();
         let mut vertex_attributes = Vec::new();
 
+        #[cfg(feature = "webgl")]
+        shader_defs.push("WEBGL".into());
+
```

Comment:
could you move that shaderdef to https://github.com/bevyengine/bevy/blob/64b0f19a6547071b54f7479deb55a5494f0fe204/crates/bevy_render/src/render_resource/pipeline_cache.rs#L187-L191?

Reply:
Yes, that makes sense.